docs/rules/order.md

```diff
@@ -285,6 +285,49 @@ import index from './';
 import sibling from './foo';
 ```
 
+### `named: true|false|{enabled: true|false, import: true|false, export: true|false, require: true|false, groups: [array]}`
+
+Enforce ordering of names within imports:
+
+ - If set to `true`, named imports must be ordered according to the `alphabetize` options
+ - If set to `false`, named imports can occur in any order
+
+`import`, `export` and `require` allows forcing ordering of named `import`s/`export`s for the corresponding kind of expression.
```

Comment:
```suggestion
`import`, `export`, and `require` allows forcing ordering of named `import`s/`export`s for the corresponding kind of expression.
```

Reply:
also, what about `exports.foo =` or `module.exports = { a, b, c }` (CJS quasi named exports)?

Reply:
I will try to add support for these as well
I totally forgot about these.

Reply:
not sure what to do about them.
Only act on `module.exports` and `exports` assignments of the shape `{ a }` and `{ a: b }`?

Reply:
i think any `exports.foo` assignment, as well as any time `module.exports` is assigned an object literal.

Reply:
This sounds a little different from what the `named` option does.
The `named` option orders import/export specifiers inside `import { … } from ""`, `export { … }`, `export { … } from ""` and `const { … } = require("")` expressions while ordering `exports.{…}` and `module.exports.{…}` assignments work on the entire file.

Do you think it might make sense to introduce a new option (something like `{ "import/order": { "exports": true } }` or whatnot) for this use case?

Reply:
`module.exports.x =` is semantically the same (or arguably, less clear) as `exports.x =`, so both of those patterns need to be recognized.

Similarly, `module.exports = { a: 1, b: 2 }` has two conceptual named exports, "a" and "b".

Reply:
That's true, though - but unlike `module.exports` or `exports`-assignments, the aforementioned expressions `const { … } = require("foo");`, `import { … } from "foo";`, `export { … } from "foo";` and `export { … }` are way more strict in their syntax

I have quite a struggle to wrap my head around the `module.exports` and `exports` aspect, tbh

Reply:
I agree those are trickier to check for - but the AST can locate a "global" variable called "module" or "exports", and we have rules that target those already, so it should be doable.

Reply:
What are your thoughts on adding an option separate from `named` for this use case?

Reply:
I think that one option for imported names, and another for exported names, would be fine - but the export one would have to include these CJS patterns.

Reply:
I will give it a try

Reply:
Alright!
I feel like it's working out properly

The new option is called `named.cjsExports`